pkg/apis/networking/types.go

```diff
@@ -361,6 +360,15 @@ type IngressRule struct {
 	// Incoming requests are matched against the host before the
 	// IngressRuleValue. If the host is unspecified, the Ingress routes all
 	// traffic based on the specified IngressRuleValue.
+	//
+	// Host can be "precise" which is an domain name without the terminating dot of a network host (e.g. "foo.bar.com")
```

Comment:
tiny nits: `an` -> `a`, and it looks like some of these lines could be wrapped to match the comments above/below - I think 80 char lines are the standard here.